Remotion/Web/Core/UI/Controls/WebLinkButton.cs

```diff
@@ -53,25 +44,42 @@ protected override void RenderContents (HtmlTextWriter writer)
       if (WcagHelper.Instance.IsWcagDebuggingEnabled() && WcagHelper.Instance.IsWaiConformanceLevelARequired())
         WcagHelper.Instance.HandleError (1, this);
 
+      Text.Write(writer);
+
       if (HasControls())
         base.RenderContents (writer);
       else
-        writer.Write (HotkeyFormatter.FormatText (_textWithHotkey!, false)); // TODO RM-8118: not null assertion
+        writer.Write (AccessKey, false); // TODO RM-8118: not null assertion
```

Comment:
I don't think the suggestion can be applied automatically, please fix in a manual commit.

```suggestion
      if (HasControls())
        base.RenderContents (writer);
      else
        Text.Write(writer);
```